.gitattributes

```diff
@@ -1,3 +1,4 @@
+/install/kubernetes/cilium/values.yaml.tmpl linguist-language=yml
```

Comment:
nice I didn't know it's possible, I manually associate file extension in Goland :1st_place_medal: